modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -58,106 +60,102 @@ public class EmailNotificationTypeTest extends BaseNotificationTypeTest {
 
-		_assertNotificationQueueEntry(
-			false,
+		_testSendNotification(
+			2,
+			ListUtil.sort(
+				Arrays.asList(
+					user1.getEmailAddress(), user2.getEmailAddress())),
+			true,
 			StringBundler.concat(
 				user1.getEmailAddress(), StringPool.COMMA,
-				user2.getEmailAddress()),
-			notificationQueueEntries.get(0));
-
-		notificationQueueEntryLocalService.deleteNotificationQueueEntry(
-			notificationQueueEntries.get(0));
-
-		notificationTemplate = _addNotificationTemplate(true);
-
-		objectActionLocalService.updateObjectAction(
-			RandomTestUtil.randomString(), objectAction.getObjectActionId(),
-			true, StringPool.BLANK, RandomTestUtil.randomString(),
-			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
-			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
-			RandomTestUtil.randomString(),
-			ObjectActionExecutorConstants.KEY_NOTIFICATION,
-			ObjectActionTriggerConstants.KEY_ON_AFTER_ADD,
-			UnicodePropertiesBuilder.put(
-				"notificationTemplateId",
-				notificationTemplate.getNotificationTemplateId()
-			).build());
-
-		_addObjectEntry();
-
-		notificationQueueEntries = ListUtil.sort(
-			notificationQueueEntryLocalService.getNotificationEntries(
-				NotificationConstants.TYPE_EMAIL,
-				NotificationQueueEntryConstants.STATUS_SENT),
-			Comparator.comparing(
-				notificationQueueEntry -> {
-					Map<String, Object> notificationRecipientSettingsMap =
-						NotificationRecipientSettingUtil.
-							getNotificationRecipientSettingsMap(
-								notificationQueueEntry);
+				user2.getEmailAddress()));
 
-					return String.valueOf(
-						notificationRecipientSettingsMap.get("to"));
-				}));
+		// Multiples emails for each main recipient, comma and space separator
 
-		Assert.assertEquals(
-			notificationQueueEntries.toString(), 2,
-			notificationQueueEntries.size());
+		_testSendNotification(
+			2,
+			ListUtil.sort(
+				Arrays.asList(
+					user1.getEmailAddress(), user2.getEmailAddress())),
+			true,
+			StringBundler.concat(
+				user1.getEmailAddress(), StringPool.COMMA_AND_SPACE,
+				user2.getEmailAddress()));
 
-		List<String> expectedToEmailAddresses = ListUtil.sort(
-			Arrays.asList(user1.getEmailAddress(), user2.getEmailAddress()));
+		// Multiples emails for each main recipient, semicolon separator
 
-		_assertNotificationQueueEntry(
-			true, expectedToEmailAddresses.get(0),
-			notificationQueueEntries.get(0));
-		_assertNotificationQueueEntry(
-			true, expectedToEmailAddresses.get(1),
-			notificationQueueEntries.get(1));
+		_testSendNotification(
+			2,
+			ListUtil.sort(
+				Arrays.asList(
+					user1.getEmailAddress(), user2.getEmailAddress())),
+			true,
+			StringBundler.concat(
+				user1.getEmailAddress(), StringPool.SEMICOLON,
+				user2.getEmailAddress()));
 
-		for (NotificationQueueEntry notificationQueueEntry :
-				notificationQueueEntries) {
+		// Multiples emails for each main recipient, with terms and comma
```

Comment:
@pedro-oliveira446 just:

```suggestion
		// Multiples emails for each main recipient, terms, comma
```